docs/output.md

```diff
@@ -2,42 +2,150 @@
+
+*Guppy* will be used to basecall and demultiplex the data. Various options have been provided to customise specific parameters and to be able to run *Guppy* on GPUs.
+
+If you have a pre-basecalled fastq file then *qcat* will be used to perform the demultiplexing if you provide the `--skip_basecalling` parameter. If you would like to skip both of these steps entirely then you can provide `--skip_basecalling --skip_demultiplexing` when running the pipeline. As a result, the structure of the output folder will depend on which steps you have chosen to run in the pipeline.
+
+*Output directories*:
+
+* `guppy/fastq/`  
+  Merged fastq output files for each barcode.
+* `guppy/basecalling/<barcode*>/`  
+  fastq output files for each barcode.
+* `guppy/basecalling/unclassified/`  
+  fastq files with reads were unassigned to any given barcode.
+* `guppy/basecalling/sequencing_summary.txt`  
+  Sequencing summary file generated by *Guppy*.
+* `guppy/basecalling/sequencing_telemetry.js`  
+  Sequencing telemetry file generated by *Guppy*.
+* `guppy/basecalling/guppy_basecaller_log-<date>.log`  
+  Log file for *Guppy* execution.
+* `qcat/fastq/<barcode*>.fastq.gz`  
+  fastq output files for each barcode.
+* `qcat/fastq/none.fastq.gz`  
+  fastq file with reads were unassigned to any given barcode.
+
+## Sequencing QC
+
+*Documentation*:  
+[PycoQC](https://github.com/a-slide/pycoQC), [NanoPlot](https://github.com/wdecoster/NanoPlot)
+
+*Description*:  
+*PycoQC* and *NanoPlot* compute metrics and generate QC plots using the sequencing summary information generated by *Guppy* e.g. distribution of read length, read length over time, number of reads per barcode and other general stats. *NanoPlot* also generates QC metrics directly from fastq files as described in the next section.
+
+![PycoQC - Number of reads per barcode](images/pycoqc_readsperbarcode.png)
+
+*Output directories*:
+
+* `pycoqc/pycoQC_output.html`  
+  `*.html` file that includes a run summary and graphical representation of various QC metrics including distribution of read length, distribution of read quality scores, mean read quality per sequence length, output per channel over experiment time and percentage of reads per barcode.  
+* `nanoplot/summary/`  
+  `*.html` files for QC metrics and individual `*.png` image files for plots.
+
+## FastQ QC
+
+*Documentation*:  
+[NanoPlot](https://github.com/wdecoster/NanoPlot), [FastQC](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/)
 
-* [FastQC](#fastqc) - read quality control
-* [MultiQC](#multiqc) - aggregate report, describing results of the whole pipeline
+*Description*:  
+*NanoPlot* can also be used to produce general quality metrics from the per-barcode fastq files generated by *Guppy* e.g. quality score distribution, read lengths and other general stats.
 
-## FastQC
+![Nanoplot - Read quality vs read length](images/nanoplot_readlengthquality.png)
```

Comment:
I guess this will have to be assessed on a case by case basis. The idea here is to just give an idea of the pipeline outputs. The description there wont be seen when the image is rendered properly anyway.